build/Build.cs

```diff
@@ -143,6 +147,68 @@ partial class Build : NukeBuild
                   .SetOutputDirectory(OutputNuget));
           }
       });
+    Target DockerLogin => _ => _
+        .Before(PushImage)
+        .Requires(() => !DockerRegistryUrl.IsNullOrEmpty())
+        .Requires(() => !DockerPassword.IsNullOrEmpty())
+        .Requires(() => !DockerUsername.IsNullOrEmpty())
```

Comment:
This `Target` will `throw` if any of the properties marked as required is null or empty.